content/en/serverless/azure_app_services/azure_app_services_container.md

```diff
@@ -9,144 +9,162 @@ further_reading:
+`DD_SERVICE` 
+: Supply a service name to be displayed in your Datadog [Service Catalog][6]. See [Unified Service Tagging][7]. 
 
-- **Logs:** If you use the [Azure integration][1], your logs are already being collected. Alternatively, you can set the `DD_LOGS_ENABLED` environment variable to `true` to capture application logs through the serverless instrumentation directly.
+`DD_ENV` 
+: A name for your environment, such as `staging` or `prod`. See [Unified Service Tagging][7].
 
-- **Custom Metrics:** You can submit custom metrics using a [DogStatsd client][3]. For monitoring Cloud Run and other serverless applications, use [distribution][8] metrics. Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][9]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
+`DD_SERVERLESS_LOG_PATH` 
+: `/home/Logfile/*.log` <br/>
+Corresponds to the path where you write your application logs. If you have changed this location, specify your custom location in this setting.
 
-- **Trace Sampling:**  To manage the APM traced request sampling rate for serverless applications, set the DD_TRACE_SAMPLE_RATE environment variable on the function to a value between 0.000 (no tracing of Container App requests) and 1.000 (trace all Container App requests).
+`DD_DOTNET_TRACER_HOME` 
+: `/datadog/tracer`
 
-Metrics are calculated based on 100% of the application’s traffic, and remain accurate regardless of any sampling configuration.
+`DD_TRACE_LOG_DIRECTORY` 
+: `/home/Logfiles/dotnet`
 
-### Environment Variables
+`CORECLR_ENABLE_PROFILING` 
+: `1`
 
-| Variable | Description |
-| -------- | ----------- |
-|`DD_API_KEY`| [Datadog API Key][6] - **Required**|
-| `DD_SITE` | [Datadog site][4] - **Required** |
-| `DD_LOGS_ENABLED` | When true, send logs (stdout and stderr) to Datadog. Defaults to false. |
-| `DD_LOGS_INJECTION`| When true, enrich all logs with trace data for supported loggers in [Java][10], [Node.js][11], [.NET][12], and [PHP][13]. See additional docs for [Python][14], [Go][15], and [Ruby][16]. |
-| `DD_TRACE_SAMPLE_RATE`|  Controls the trace ingestion sample rate `0.0` and `1.0`|
-| `DD_SERVICE`      | See [Unified Service Tagging][5].                                       |
-| `DD_VERSION`      | See [Unified Service Tagging][5].                                       |
-| `DD_ENV`          | See [Unified Service Tagging][5].                                       |
-| `DD_SOURCE`       | See [Unified Service Tagging][5].                                       |
-| `DD_TAGS`         | See [Unified Service Tagging][5].                                       |
+`CORECLR_PROFILER` 
```

Comment:
@jcstorms1 do you think we should move this into the dockerfile if the values are static?

Reply:
Yes. Lines 124-134 configures the .NET tracer and is .NET specific.  I assume we don't want to support a table of environment variables per runtime, so it would be better to just put these in the Dockerfile.